### PR TITLE
PC-1348

### PR DESCRIPTION
**Pull request by eyalvo-matrix**

### CODE REVIEW

File: modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql [hunk 1]
```diff
@@ -297,3 +297,31 @@ VALUES
...
+#IfNotRow2D questionnaires_schemas form_name form_diagnosis_and_recommendations_questionnaire question Decision
+INSERT INTO `questionnaires_schemas` (`qid`, `form_name`,`form_table`, `column_type`, `question`)
+VALUES
+('1', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Findings details'),
+('2', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Diagnosis details'),
+('3', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Treatment details'),
+('4', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Instructions for further treatment'),
+('5', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Decision'),
+('6', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Evacuation way'),
+('7', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Sick leave');
```
Comment (matrix-amiel) [inline modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql]: why Sick leave is not an integer?

File: modules/EmergencyMedicine/sql/install.sql [hunk 2]
```diff
@@ -238,3 +238,27 @@ VALUES
...
+
+INSERT INTO `questionnaires_schemas` (`qid`, `form_name`,`form_table`, `column_type`, `question`)
+VALUES
+('1', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Findings details'),
+('2', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Diagnosis details'),
+('3', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Treatment details'),
+('4', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Instructions for further treatment'),
+('5', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Decision'),
+('6', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Evacuation way'),
+('7', 'diagnosis_and_recommendations_questionnaire','form_diagnosis_and_recommendations_questionnaire', 'string', 'Sick leave');
```
Comment (matrix-amiel) [inline modules/EmergencyMedicine/sql/install.sql]: why Sick leave is not an integer?

File: modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql (hunk 1 above)
Comment (eyalvo-matrix) [reply modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql]: will fix that

File: modules/EmergencyMedicine/sql/install.sql (hunk 2 above)
Comment (eyalvo-matrix) [reply modules/EmergencyMedicine/sql/install.sql]: will fix that

**Pull request merged**